build.sbt

```diff
@@ -33,46 +35,46 @@ lazy val core = Project("elastic4s-core", file("elastic4s-core"))
-      "org.apache.lucene"                     % "lucene-spatial3d"          % LuceneVersion,
-      "org.apache.lucene"                     % "lucene-suggest"            % LuceneVersion,
-      "org.elasticsearch.client"              % "transport"                 % ElasticsearchVersion,
-      "org.apache.lucene"                     % "lucene-join"               % LuceneVersion,
-      "org.apache.logging.log4j"              % "log4j-api"                 % Log4jVersion,
-      "org.apache.logging.log4j"              % "log4j-core"                % Log4jVersion,
-      "org.apache.logging.log4j"              % "log4j-1.2-api"             % Log4jVersion,
-      "org.apache.logging.log4j"              % "log4j-slf4j-impl"          % Log4jVersion,
-      "com.carrotsearch"                      % "hppc"                      % "0.7.1",
-      "joda-time"                             % "joda-time"                 % "2.9.9",
-      "com.fasterxml.jackson.core"            % "jackson-core"              % JacksonVersion,
-      "com.tdunning"                          % "t-digest"                  % "3.1"
+      "org.elasticsearch" % "elasticsearch" % ElasticsearchVersion,
+      "org.locationtech.spatial4j" % "spatial4j" % "0.6",
+      "com.vividsolutions" % "jts" % "1.13",
+      "io.netty" % "netty-all" % "4.1.10.Final",
+      "org.apache.lucene" % "lucene-core" % LuceneVersion,
+      "org.apache.lucene" % "lucene-analyzers-common" % LuceneVersion,
+      "org.apache.lucene" % "lucene-backward-codecs" % LuceneVersion,
+      "org.apache.lucene" % "lucene-grouping" % LuceneVersion,
+      "org.apache.lucene" % "lucene-highlighter" % LuceneVersion,
+      "org.apache.lucene" % "lucene-join" % LuceneVersion,
+      "org.apache.lucene" % "lucene-memory" % LuceneVersion,
+      "org.apache.lucene" % "lucene-misc" % LuceneVersion,
+      "org.apache.lucene" % "lucene-queries" % LuceneVersion,
+      "org.apache.lucene" % "lucene-queryparser" % LuceneVersion,
+      "org.apache.lucene" % "lucene-sandbox" % LuceneVersion,
+      "org.apache.lucene" % "lucene-spatial" % LuceneVersion,
+      "org.apache.lucene" % "lucene-spatial-extras" % LuceneVersion,
+      "org.apache.lucene" % "lucene-spatial3d" % LuceneVersion,
+      "org.apache.lucene" % "lucene-suggest" % LuceneVersion,
+      "org.elasticsearch.client" % "transport" % ElasticsearchVersion,
+      "org.apache.lucene" % "lucene-join" % LuceneVersion,
+      "org.apache.logging.log4j" % "log4j-api" % Log4jVersion,
+      "org.apache.logging.log4j" % "log4j-core" % Log4jVersion,
+      "org.apache.logging.log4j" % "log4j-1.2-api" % Log4jVersion,
+      "org.apache.logging.log4j" % "log4j-slf4j-impl" % Log4jVersion,
+      "com.carrotsearch" % "hppc" % "0.7.1",
+      "joda-time" % "joda-time" % "2.9.9",
+      "com.fasterxml.jackson.core" % "jackson-core" % JacksonVersion,
+      "com.tdunning" % "t-digest" % "3.1"
     )
```

Comment:
It's not much of a formatter since it's broken all my nicely aligned deps :)

Reply:
We can do `align=more`, by default it's `align=some`
See: http://scalameta.org/scalafmt/#align=more

Reply:
Please